docs/handlers/python.md

```diff
@@ -317,3 +318,55 @@ td p {
   margin-bottom: 0 !important;
 }
 ```
+
+## Recommended style (ReadTheDocs)
```

Comment:
(The problem is not new in this pull request, but here goes anyway:)

Could you make this CSS available as a file, or perhaps even include it by default somehow? Otherwise it seems like users are forced to copy-paste this into their project, with the risk of it getting outdated

Reply:
Those are valid points. I guess I was just a bit reluctant to incorporate the CSS into the package since I suck at it :stuck_out_tongue: 

Hopefully if my CSS is bad, users will fix it in PRs, so why not! I'll see how it can be done.

Reply:
Opened #189 to track this.